src/protocol/bolt/src/test/groovy/org/apache/jmeter/protocol/bolt/sampler/BoltSamplerSpec.groovy

```diff
@@ -44,10 +45,11 @@ class BoltSamplerSpec extends Specification {
         def variables = new JMeterVariables()
         // ugly but could not find a better way to pass the driver to the sampler...
         variables.putObject(BoltConnectionElement.BOLT_CONNECTION, driver)
+        variables.putObject(BoltConnectionElement.DATABASE, BoltConnectionElement.DATABASE)
```

Comment:
The change is not covered by this test. There should be additional tests to check if the selection of a db is actually taken into account when opening the session